### PR TITLE
Blog entity

### PR DESCRIPTION
**Pull request by Kusken**
Please make a thorough review. Alot depends on this code.

### CODE REVIEW

File: OpenBlog/OpenBlog.Persistence/DataModel/Id/Id.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ public Id()
 
         public Id(string id)
         {
-            if (id == Guid.Empty.ToString(Specifier))
+            if (id.IsEmptyGuid(Specifier))
                 throw new FormatException();
```
Comment (ooleoole) [inline OpenBlog/OpenBlog.Persistence/DataModel/Id/Id.cs]: add a message to ex maybe?

Comment (ooleoole) [inline OpenBlog/OpenBlog.Persistence/DataModel/Id/Id.cs]: add a message to ex?

Comment (Kusken) [reply OpenBlog/OpenBlog.Persistence/DataModel/Id/Id.cs]: Will do!

**Comment by Kusken** [comment]
Will add comments for all the exception messages and then make a new pull request.

**Pull request closed without merging**